### PR TITLE
feat: APP-347 get rid of buy buttons and sell orders page in main nav

### PR DESCRIPTION
**Pull request by r41ph**
## Description

https://regennetwork.atlassian.net/browse/APP-347

---

### Author Checklist

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test
https://deploy-preview-2488--regen-marketplace.netlify.app/storefront

1. Verify that the Trade item in the main navigation is not present.
2. When _logged in_ go to `/storefront` and check that the Buy button is no present in the Actions column but the Cancel option is still available.
3. When _not logged in_ go to `/storefront` and check that the Actions column is not visible.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by r41ph** [comment]
@erika see testing instructions

File: web-marketplace/src/pages/Marketplace/Storefront/Storefront.tsx [hunk 1]
```diff
@@ -256,62 +242,30 @@ export const Storefront = (): JSX.Element => {
...
+                                },
+                              },
+                            ]}
+                            sx={{ width: '100%' }}
+                          />
+                        )
+                      );
+                    }
+                  : undefined
+              }
```
Comment (blushi) [inline web-marketplace/src/pages/Marketplace/Storefront/Storefront.tsx:268]: we can also get rid of `BuyCreditsModal` and related code: `isBuyModalOpen`, etc.

**Review by blushi — APPROVED** [review]
pre-approving, pending some remaining clean up

File: web-marketplace/src/pages/Marketplace/Storefront/Storefront.tsx (hunk 1 above)
Comment (r41ph) [reply web-marketplace/src/pages/Marketplace/Storefront/Storefront.tsx:268]: Thanks for pointing that out

**Comment by erikalogie** [comment]
This looks good, I would just get rid of this shadow when there are no actions in the logged out version

Monosnap Regen Marketplace 2024-10-07 09-45-00

**Comment by r41ph** [comment]
> This looks good, I would just get rid of this shadow when there are no actions in the logged out version

Ah ok I'll remove it. I thought that was part of the design, like a 'hint' for letting users know there's more table to scroll horizontally.

**Comment by r41ph** [comment]
@erikalogie I have removed the shadow, please have a look

**Comment by erikalogie** [comment]
LGTM!

**Pull request merged**